src/main/examples/src/test/java/com/pushtechnology/diffusion/maven/plugin/examples/DiffusionConnectIT.java

```diff
@@ -66,19 +71,29 @@ public void testSession() throws Exception {
         assertNotNull(session.getSessionId());
 
         final Topics topics = session.feature(Topics.class);
+        final TopicControl topicControl = session.feature(TopicControl.class);
+        final TopicUpdate topicUpdate = session.feature(TopicUpdate.class);
+        final CompletableFuture<TopicControl.AddTopicResult> future = topicControl.addTopic(
+            "foo", TopicType.STRING);
 
-        topics.addFallbackTopicStream(topicStream);
-        topics.subscribe(">Diffusion", callback);
+        // Wait for the CompletableFuture to complete
+        future.get(10, TimeUnit.SECONDS);
+
+        topics.addStream(">foo", String.class, topicStream);
```

Comment:
You've selectors that use the path prefix and ones that don't. I'd say "foo" throughout.